Maxim Borodin. Task 3 & Task 4

maximborodin opened the pull request:
Ultra fast storage

maximborodin committed:
First commit

maximborodin committed:
CheckStyle

maximborodin committed:
Checkstyle Fixed

maximborodin committed:
Fixed Kozlov's pom.(((

maximborodin committed:
Fixed readKeys() and closing

maximborodin committed:
Fixed something

maximborodin committed:
Checkstyle

maximborodin committed:
Added setLength(0) in writeTemporaryStorage()

maximborodin committed:
Removed DataOutputStream from writeTemporaryStorage

maximborodin committed:
Fixed problem in write() function (sometimes data haven't saved in keys() and toWriteStorage

maximborodin committed:
Constant's regrowth (1500), first was 500

maximborodin committed:
Constant's regrowth (100)

maximborodin committed:
Merge remote-tracking branch 'upstream/master'
:w

malchun reviewed (commented):
file lock отсутствует

maximborodin committed:
Added Lock and hash

maximborodin committed:
Deleted something from Kryloff and codestyle

maximborodin committed:
Deleted Kryloff test_tasks

maximborodin committed:
Merge remote-tracking branch 'upstream/master'

maximborodin committed:
Started dealing with 4th

maximborodin committed:
Added Database:

maximborodin committed:
Merge remote-tracking branch 'upstream/master'

maximborodin committed:
Added clearing calculator in Task1, database added to ignored files

maximborodin committed:
Changes in pom

maximborodin committed:
Codestyle

maximborodin committed:
Added authorization

maximborodin committed:
Removed 8/5000
:wq
:wq
Removed unnecessary dependency in pom and fixed problem with no authentification

maximborodin committed:
Removed unnecessary dependency in pom and fixed trouble with No Auth

maximborodin committed:
Merge branch 'master' of https://github.com/maximborodin/mipt-java-2016
